web-console/src/lib/services/pipelineManager.ts

```diff
@@ -300,13 +301,13 @@ const consolidatePipelineStatus = (
     .with(['Shutdown', P.any, P.nullish, { SystemError: P.select() }], (SystemError) => ({
       SystemError
     }))
-    .with(['Shutdown', 'Running', P.any, P._], () => 'Starting up' as const) // Workaround when fetching status right after POST start action
-    .with(['Provisioning', 'Running', P.any, P._], () => 'Starting up' as const) // Workaround when fetching status right after POST start action
-    .with(['Shutdown', 'Paused', P.any, P._], () => 'Starting up' as const) // Workaround when fetching status right after POST start_paused action
+    .with(['Shutdown', 'Running', P.any, P._], () => 'Provisioning' as const) // Workaround when fetching status right after POST start action
+    .with(['Provisioning', 'Running', P.any, P._], () => 'Provisioning' as const) // Workaround when fetching status right after POST start action
+    .with(['Shutdown', 'Paused', P.any, P._], () => 'Provisioning' as const) // Workaround when fetching status right after POST start_paused action
```

Comment:
It would be preferable this (deployment_status=Shutdown, deployment_desired_status=Running/Paused, program_status=Any) would be named differently, as it is not provisioning. How about "Preparing"?